common/src/api/external/mod.rs

```diff
@@ -753,9 +753,12 @@ pub struct Instance {
 pub struct Disk {
     #[serde(flatten)]
     pub identity: IdentityMetadata,
+    // TODO these should be names!
     pub project_id: Uuid,
     pub snapshot_id: Option<Uuid>,
+    pub image_id: Option<Uuid>,
```

Comment:
I'm not actually sure that they need to be. I know that's confusing given that we tend to do most of our lookups based on names, but if you look at `Instance`, `RouterRoute`, `VpcFirewallRule`, `NetworkInterface`, etc they're all returning ids. I think once @davepacheco wraps up #798 it'll be relatively straight forward to add the `/by_id/` routes and the id references here would likely be preferred due to their stability.

Reply:
cool, removed in 88f23c5e